tests/unit/Claim/ClaimsTest.php

```diff
@@ -86,381 +86,384 @@ public function constructorErrorProvider() {
+		$statement1 = $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P15' ) ) );
+		$statement2 = $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P16' ) ) );
 
-		$claims->addClaim( $claim1 );
-		$claims->addClaim( $claim2 );
+		$statements->addClaim( $statement1 );
+		$statements->addClaim( $statement2 );
 
-		$this->assertSame( 2, $claims->count() );
-		$this->assertEquals( $claim1, $claims[$claim1->getGuid()] );
-		$this->assertEquals( $claim2, $claims[$claim2->getGuid()] );
+		$this->assertSame( 2, $statements->count() );
+		$this->assertEquals( $statement1, $statements[$statement1->getGuid()] );
+		$this->assertEquals( $statement2, $statements[$statement2->getGuid()] );
 
-		$claims->addClaim( $claim1 );
-		$claims->addClaim( $claim2 );
+		$statements->addClaim( $statement1 );
+		$statements->addClaim( $statement2 );
 
-		$this->assertSame( 2, $claims->count() );
+		$this->assertSame( 2, $statements->count() );
 
-		$this->assertNotNull( $claims->getClaimWithGuid( $claim1->getGuid() ) );
-		$this->assertNotNull( $claims->getClaimWithGuid( $claim2->getGuid() ) );
+		$this->assertNotNull( $statements->getClaimWithGuid( $statement1->getGuid() ) );
+		$this->assertNotNull( $statements->getClaimWithGuid( $statement2->getGuid() ) );
 
 		// Insert claim at the beginning:
-		$claim3 = $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P17' ) ) );
-		$claims->addClaim( $claim3, 0 );
-		$this->assertEquals( 0, $claims->indexOf( $claim3 ), 'Inserting claim at the beginning failed' );
+		$statement3 = $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P17' ) ) );
+		$statements->addClaim( $statement3, 0 );
+		$this->assertEquals( 0, $statements->indexOf( $statement3 ), 'Inserting statement at the beginning failed' );
 
 		// Insert claim at another index:
-		$claim4 = $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P18' ) ) );
-		$claims->addClaim( $claim4, 1 );
-		$this->assertEquals( 1, $claims->indexOf( $claim4 ), 'Inserting claim at index 1 failed' );
+		$statement4 = $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P18' ) ) );
+		$statements->addClaim( $statement4, 1 );
+		$this->assertEquals( 1, $statements->indexOf( $statement4 ), 'Inserting statement at index 1 failed' );
 
 		// Insert claim with an index out of bounds:
-		$claim5 = $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P19' ) ) );
-		$claims->addClaim( $claim5, 99999 );
-		$this->assertEquals( 4, $claims->indexOf( $claim5 ), 'Appending claim by specifying an index out of bounds failed' );
+		$statement5 = $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P19' ) ) );
+		$statements->addClaim( $statement5, 99999 );
+		$this->assertEquals( 4,
+			$statements->indexOf( $statement5 ),
+			'Appending statement by specifying an index out of bounds failed'
+		);
 	}
 
 	public function testIndexOf() {
-		$claims = new Claims();
-		$claimArray = array(
+		$statements = new Claims();
```

Comment:
All the variable name changes here don't make much sense if `$statements` still is of type `Claims` and `$statementArray` still is an array of `makeClaim` calls.